src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -618,6 +618,11 @@ public override NamedTypeSymbol TupleUnderlyingType
             }
         }
 
+        internal override TypeSymbol ExtensionClassTypeNoUseSiteDiagnostics
+        {
+            get { return _underlyingType.ExtensionClassTypeNoUseSiteDiagnostics; }
```

Comment:
Can `_underlyingType` be an extension class? If not, consider asserting instead.